18F/identity-idp
# LG-11454: Add rename support for face or touch unlock

## 🎫 Ticket

[LG-11454](https://cm-jira.usa.gov/browse/LG-11454)

## 🛠 Summary of changes

Adds the option for a user to manage (rename or delete) a Face or Touch Unlock authenticator from within the account dashboard, managed inline without the need to navigate to a separate screen.

## 📜 Testing Plan

1. (Prerequisite) Have an account with Face or Touch Unlock
2. Go to http://localhost:3000
3. Sign in
4. On account dashboard, click "Manage" for Face or Touch Unlock
5. Observe that you are able to manage the authenticator inline
6. Disable JavaScript and repeat steps 4 and 5, verifying that there is feature parity in the no-JavaScript experience

## 👀 Screenshots

https://github.com/18F/identity-idp/assets/1779930/d9a47290-589b-413a-8c97-47bbfa40929e
